package.json

```diff
@@ -14,11 +14,11 @@
     "start:debug": "nest start --debug --watch",
     "start:prod": "node dist/main",
     "lint": "tslint -c tslint.json --project tsconfig.json",
-    "test": "jest",
+    "test": "jest  --forceExit --detectOpenHandles",
```

Comment:
Why do we need these flags? The [docs](https://jestjs.io/docs/en/cli#--detectopenhandles) state detectOpenHandles "has a significant performance penalty and should only be used for debugging"

Reply:
It was because a bug in nestjs closing some connections when you use typeOrm